src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala

```diff
@@ -8,6 +8,7 @@ import beam.agentsim.agents._
 import beam.agentsim.agents.household.HouseholdActor.{MobilityStatusInquiry, MobilityStatusResponse, ReleaseVehicle}
 import beam.agentsim.agents.modalbehaviors.ChoosesMode._
 import beam.agentsim.agents.modalbehaviors.DrivesVehicle.{ActualVehicle, Token, VehicleOrToken}
+import beam.agentsim.agents.planning.Strategy.ModeChoiceStrategy
```

Comment:
Which change takes precedence, this one or https://github.com/LBNL-UCB-STI/beam/pull/3476?